pinot-core/src/main/java/com/linkedin/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -203,6 +203,13 @@ public BrokerResponseNative reduceOnDataTable(@Nonnull BrokerRequest brokerReque
           setGroupByHavingResults(brokerResponseNative, aggregationFunctions, aggregationFunctionSelectStatus,
               brokerRequest.getGroupBy(), dataTableMap, brokerRequest.getHavingFilterQuery(),
               brokerRequest.getHavingFilterSubQueryMap());
+          if (brokerMetrics != null) {
+            long groupBySize = 0L;
```

Comment:
Should be int right?

Reply:
That's because the type of parameter `unitCount` is long according to the method signature, i.e. `public void addMeteredTableValue(final String tableName, final M meter, final long unitCount)`